docs/green-lanes.md

```diff
@@ -0,0 +1,79 @@
+# Windsor Framework - Green Lanes
+
+The green lanes support exists to allow categorisation of Goods Nomenclatures according to the Windsor Framework agreement.
+
+The categorisation determines whether a good can be moved between GB and NI under the Windsor Framework. There are three potential categorisations 1, 2 and Standard, with 1 being the most restrictive and Standard being the least.
+
+**Note:** All green lanes specific code is namespaced under the `GreenLanes::` namespace
+
+## Database
+
+### Themes
+
+We have a simple type table with each of the themes from Windsor Framework held in its own row.
+
+This provides both the category and a description of the Theme.
+
+### Category Assessments
+
+The categorisation itself is derived from the measures applied to a Goods Nomenclature. To categorise we need to match particular measures with the relevant Windsor Framework Theme.
+
+We have a join model - `CategoryAssessment` which matches particular measures on `measure_type_id`, `measure_generating_regulation_id` and `measure_generating_regulation_role`. This model the in turn points to the relevant `Theme` record which provides the category
```

Comment:
I'd avoid using "We have", and prefer a direct speech:

The model `CategoryAssessment` matches particular measures on ...
---
This model the in turn ...
something wrong in this phrase. Maybe "the" should be removed.
Also, what is "this model", does it refer to CategoryAssessment?